Yoast/YoastSEO.js
# Add Polish prominent words

## Summary

This PR can be summarized in the following changelog entry: Adds prominent words for Polish (props @tmotyl and @oskardydo from @macopedia )


## Test instructions

This PR can be tested by following these steps:

* Run grunt build:js.
* Use the prominent word example in your browser.
* Set locale to pl_PL.
* Paste a Polish text into the example.
* Make sure you see no strange word combinations, counts, etc.

Fixes #1530
